### PR TITLE
Set batch options for internal sync

### PR DESCRIPTION
**Pull request by smithsz**
Allows an operator to set batch_size and batch_count for internal push replications to a specific node via config:set.

### CODE REVIEW

**Comment by theburge** [comment]
@smithsz:
I'd be inclined in ignore the edge case, and defer the behaviour to the underlying code (i.e., the config setting should get the value in there, the business end of the code should interpret the value).

For `s:repl_dbs/1`, perhaps a neater choice is to copy the pattern elsewhere and introduce `s:repl_dbs/2` with an `RepOpts` parameter that gets passed through, then make `s:repl_dbs/1` call the `/2` version.

**Comment by smithsz** [comment]
This should help with two operational tasks:

- Configuring how hard we push data to another cluster node
- Stop internal rep on a single cluster node (maybe useful when compactions aren't flipping)

*Some examples:*
```
(node1@127.0.0.1) 1> % stop node1 from pushing data to node3
(node1@127.0.0.1) 1> config:set("mem3.batch_size", "node3@127.0.0.1", "0").
ok
(node1@127.0.0.1) 2> % quickly sync data to node3
(node1@127.0.0.1) 2> config:set("mem3.batch_count", "node3@127.0.0.1", "3").
ok
(node1@127.0.0.1) 3> config:set("mem3.batch_size", "node3@127.0.0.1", "200").
ok
```

File: src/mem3_rep.erl [hunk 1]
```diff
@@ -42,16 +42,24 @@ go(DbName, Node, Opts) when is_binary(DbName), is_atom(Node) ->
...
 
 
-go(#shard{} = Source, #shard{} = Target, Opts) ->
+go(#shard{} = Source, #shard{node=TargetNode} = Target, Opts) ->
     mem3_sync_security:maybe_sync(Source, Target),
     BatchSize = case proplists:get_value(batch_size, Opts) of
         BS when is_integer(BS), BS > 0 -> BS;
-        _ -> 100
+        _ ->
+            try list_to_integer(config:get("mem3.batch_size", atom_to_list(TargetNode), "100"))
```
Comment (iilyak) [inline src/mem3_rep.erl]: You could use config:get_integer/3 here.

**Comment by smithsz** [comment]
@iilyak - I missed a trick there. Thanks!

**Comment by smithsz** [comment]
+1

**Comment by davisp** [comment]
And I'll +1 from my own machine as well.

**Pull request merged**